content/en/blog/_posts/2024-03-08-mid-cycle-1.30.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: blog
+title: 'Kubernetes Removals and Major Changes in Kubernetes 1.30'
+date: 2024-03-08
+slug: kubernetes-1-30-upcoming-changes
+---
+
+**Authors:** Amit Dsouza, Frederick Kautz, Kristin Martin, Abigail McCarthy, Natali Vlatko 
+
+As Kubernetes grows and matures, features may be deprecated, removed, or replaced with improvements for the health of the project. Kubernetes v1.30 includes several major changes and one removal.
+
+## The Kubernetes API Removal and Deprecation process
+
+The Kubernetes project has a [well-documented deprecation policy](/docs/reference/using-api/deprecation-policy/) for features. This policy states that stable APIs may only be deprecated when a newer, stable version of that same API is available and that APIs have a minimum lifetime for each stability level. A deprecated API is one that has been marked for removal in a future Kubernetes release; it will continue to function until removal (at least one year from the deprecation), but usage will result in a warning being displayed. Removed APIs are no longer available in the current version, at which point you must migrate to using the replacement.
+
+* Generally available (GA) or stable API versions may be marked as deprecated, but must not be removed within a major version of Kubernetes.
+* Beta or pre-release API versions must be supported for 3 releases after deprecation.
+* Alpha or experimental API versions may be removed in any release without prior deprecation notice.
+
+Whether an API is removed as a result of a feature graduating from beta to stable or because that API simply did not succeed, all removals comply with this deprecation policy. Whenever an API is removed, migration options are communicated in the documentation.
+
+## Removal for Kubernetes v1.30
+
+### Removal of `SecurityContextDeny` admission plugin
+
+The `SecurityContextDeny` admission plugin, deprecated since v1.27, is being [removed in v1.30](https://github.com/kubernetes/kubernetes/pull/122612). The `PodSecurity` admission plugin, [available since v1.25](/docs/concepts/security/pod-security-admission/), is recommended instead. See [Admission Controllers reference](/docs/reference/access-authn-authz/admission-controllers/#securitycontextdeny) for more information. 
```

Comment:
How about dropping this, and focusing on the “upcoming changes” aspect only? The actual removal part is quite a trivial one.